sources/HUBViewControllerImplementation.m

```diff
@@ -48,6 +48,8 @@
 #import "HUBActionHandler.h"
 #import "HUBViewModelDiff.h"
 
+CFTimeInterval const kDownloadingTimeTreshold = 0.07;
```

Comment:
I agree that it probably makes sense to expose this value via some header.

If not we need to add `static` here to not pollute the global namespace, like:
```objc
static CFTimeInterval  const HUBDownloadingTimeThreshold = 0.07;
```